### DISCUSSION TITLE
Failed to fetch sample from MONAI Label Server

### DISCUSSION THREAD

**Discussion by aleoneshahvar** [Project-MONAI/MONAILabel#502] [Q&A] [answered]
I successfully imported to Monai label server to 3D slicer and have loaded the model however when I try to grab a sample, it gives this error: 
Traceback (most recent call last):

File "C:/Users/Ariela/AppData/Local/NA-MIC/Slicer 4.13.0-2021-10-03/NA-MIC/Extensions-30282/MONAILabel/lib/Slicer-4.13/qt-scripted-modules/MONAILabel.py", line 1359, in onNextSampleButton

if self.samples.get(sample["id"]) is not None:

KeyError: 'id'

I have tried to compare my file setup to that of the Youtube tutorial on Monai Label that was recently posted but I do not see any elements missing. Please let me know if you have any suggestions on how I can fix this error!

**Comment by SachidanandAlle** [comment, marked as answer]
Do you have images in the studies folder?

**Reply by aleoneshahvar** [reply]
I have them in a folder under my documents and when I load the model I use -s followed by the path to that folder, is this what you mean by studies folder?

**Reply by aleoneshahvar** [reply]
I was able to resolve this issue by converting the DICOM images to nii.gz!
